Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
```

Comment:
why we need a new Chart.yaml. Don't we have this already in https://github.com/Mellanox/network-operator/blob/master/deployment/network-operator/Chart.yaml

Reply:
It was a part of WIP patch. I'll remove it